tests/types/test_schema.py

```diff
@@ -251,6 +252,21 @@ def test_get_tensor_shape(dict_of_ndarrays):
         _infer_schema({"x": 1})
 
 
+@pytest.fixture
+def dict_of_sparse_matrix():
+    return {
+        "csc": csc_matrix(np.arange(0, 12, 0.5).reshape(3, 8)),
+        "csr": csr_matrix(np.arange(0, 12, 0.5).reshape(3, 8)),
+    }
```

Comment:
If we don't use keys in `test_get_sparse_matrix_data_type_and_shape`, can we just use a list rather than a dict?

Reply:
I would like to preserve the key, in future if we need distinguish csr or csc in test, we can use the key

Reply:
Got it.